docs/references/subcommands/analyze.md

```diff
@@ -24,7 +24,8 @@ In addition to the [usual FOSSA project flags](#common-fossa-project-flags) supp
 | `--policy 'some policy'`              |       | Assign a specific FOSSA policy to this project. Mutually excludes `--policy-id`.    |
 | `--policy-id 'some policy id'`        |       | Assign a specific FOSSA policy to this project by id. Mutually excludes `--policy`. |
 | `--project-label`                     |       | assign up to 5 labels to the project                                                |
-| `--release-group-name`                |       | the name of the release group to add this project to                                |
+| `--release-group-name 'MY_RG'`        |       | add the project to this release group (also requires `--release-group-release`)     |
```

Comment:
[optional] Can you add example in FAQ section? This is slightly involved option, so would be good to have something that can be explained in detail and copy-pastable example

Reply:
done, thanks for the suggestion! https://github.com/fossas/fossa-cli/edit/ryanlink-analyze-subcommand-update/docs/references/subcommands/analyze.md?pr=%2Ffossas%2Ffossa-cli%2Fpull%2F1415#how-do-i-add-a-project-to-a-release-group-when-it-is-analyzed